### PR TITLE
Rework building of many linux python packages with GitHub actions.

### LINKED ISSUE #860: Provide PyPI distribution for Python3.9/Linux
The new library looks helpful. We'd be interested to integrate it into a library for working with SED-ML, including validating SED-ML files that describes simulations of CellML models. From your documentation, this should be easy for us to do.

Could you provide a distribution for Python 3.9/Linux? Currently there's Python 3.7 and 3.8 for Linux and 3.9 for Mac and Windows.

### PR DESCRIPTION
**Pull request by hsorby**
Adding in a Python 3.9 variant.

### CODE REVIEW

**Comment by hsorby** [comment]
Addresses #860.

File: .github/workflows/deploy-on-release.yml [hunk 1]
```diff
@@ -28,182 +32,116 @@ jobs:
...
         # fetch libraries
-        git clone https://github.com/cellml/libcellml -b release --depth 1 $src_dir/libcellml
-        git clone https://github.com/OpenCMISS-Dependencies/zlib -b v1.2.3 --depth 1 $src_dir/zlib
-        git clone https://github.com/OpenCMISS-Dependencies/libxml2 -b v2.9.6 --depth 1 $src_dir/libxml2
-
-        curl -L https://cfhcable.dl.sourceforge.net/project/swig/swig/swig-4.0.2/swig-4.0.2.tar.gz > $downloads_dir/swig-4.0.2.tar.gz
-        curl -L ${{ matrix.config.python_tarball }} > $downloads_dir/Python.tar.gz
-        curl -L https://www.openssl.org/source/openssl-1.1.1h.tar.gz > $downloads_dir/openssl-1.1.1h.tar.gz
-        curl -L ftp://sourceware.org/pub/libffi/libffi-3.2.1.tar.gz > $downloads_dir/libffi-3.2.1.tar.gz
+        git clone https://github.com/hsorby/l1bcellml -b develop --depth 1 $SRC_DIR/libcellml
```
Comment (nickerso) [inline .github/workflows/deploy-on-release.yml]: should this be `cellml/libcellml` (i.e., the prime libcellml repo)?

Comment (hsorby) [reply .github/workflows/deploy-on-release.yml]: Definitely.

**Review by nickerso — APPROVED** [review]
assuming tests still pass, this looks good to me.

**Review by agarny — APPROVED** [review]

**Pull request merged**